src/Agent.Worker/TaskManager.cs

```diff
@@ -38,7 +84,10 @@ public sealed class TaskManager : AgentService, ITaskManager
                 return;
             }
 
-            Trace.Info("Getting task.");
+            // Inform the user that a download is taking place. The download could take a while if
+            // the task zip is large. It would be nice to print the localized name, but it is not
+            // available from the reference included in the job message.
+            executionContext.Output(StringUtil.Loc("DownloadingTask0", task.Name));
```

Comment:
I don't mind adding it if you think it adds value. I didn't add it because it's in the diag log already, and I don't think it's useful for the customer (in the build log).